docs/glossary.mdx

```diff
@@ -0,0 +1,94 @@
+
+### Digest
+
+A [digest](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests) is a property of a Descriptor which uniquely identifies content by taking a collision-resistant hash of the bytes.
+
+### Distribution-spec
+
+The [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec/blob/main/spec.md) defines an API protocol to facilitate and standardize the distribution of content.
+
+### Extensions
+
+The Distribution Specification Project includes a process and API for prototyping and testing [extensions](https://github.com/opencontainers/distribution-spec/blob/main/extensions/README.md) to the Distribution API.
+
+### Image Layout
+
+The [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) is the directory structure for OCI content-addressable blobs and location-addressable references (refs). It must contain a `blobs` directory, `oci-layout` file and an `index.json` file. To learn more, check out the how-to guide on OCI Layouts.
+
+### Image Manifests
+
+An [image manifest](https://github.com/opencontainers/image-spec/blob/main/manifest.md#image-manifest) provides a configuration and set of layers for a single container image for a specific architecture and operating system.
+
+### Image-spec
+
+The [OCI Image Spec](https://github.com/opencontainers/image-spec/blob/main/spec.md) defines an OCI Image, consisting of an image manifest, an image index (optional), a set of filesystem layers, and a configuration.
+
+### Image Index Specification
+
+The [image index](https://github.com/opencontainers/image-spec/blob/main/image-index.md) is a higher-level manifest which points to specific image manifests, ideal for one or more platforms. It is a multi-descriptor entry point.
+
+### Local Registry
+
+A registry is a place where container images and artifacts can easily be stored and accessed. Whereas, a local registry (like [zot](https://zotregistry.io/)) is a registry that is present on our local machine. You can follow our [quick start guide](./quickstart.mdx) if you would like to try using zot registry.
+
+### Manifest Referrers API
+
+Artifact-manifest provides the ability to reference artifacts to existing artifacts. Reference artifacts include signatures, SBoMs and many other types. The [manifest `referrers` API](https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md) returns all artifacts that have a `subject` of the given manifest digest.
+
+### Referrer API
+
+The Referrers API returns a list of manifests that reference a blob. You can understand how to use this API by referring to the [details](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#listing-referrers) given in the distribution spec. 
+
+### Referrer Tag Schema
+
+The Referrer Tag Schema helps to pull the current list of referrers based on different digest formats or tags.
```

Comment:
I'm not entirely sure here, but doesn't the schema define the supported format of digests or tags?

Reply:
@TerryHowe accidentally missed this comment. That is what I understood after reading the content of opencontainers, however, I may have misinterpreted it.

Reply:
Hope this might help  - https://github.com/oras-project/oras-www/pull/240